src/black/report.py

```diff
@@ -55,7 +62,7 @@ def failed(self, src: Path, message: str) -> None:
 
     def path_ignored(self, path: Path, message: str) -> None:
         if self.verbose:
-            out(f"{path} ignored: {message}", bold=False)
+            out(f"{path} ignored: {message}", style=OutputLevels.warning)
```

Comment:
I feel like this should be some other level, like `info`.

Reply:
I wanted to highlight them, I could use dim yellow though, by changing to `OutputLevels.info` and `fg="yellow"`.

Reply:
Any particular reason as to why you'd want to highlight them? Don't get me wrong, it could be nice. But this is why we define the styles right? 😄

Reply:
I just wanted them to be easily recognizable from the crowd of "wasn't modified logs".